opts/opts.go

```diff
@@ -182,27 +212,33 @@ func ValidateDevice(val string) (string, error) {
 //    [host-dir:]container-path[:rw|ro]
 // It will also validate the mount mode.
 func ValidatePath(val string) (string, error) {
-	return validatePath(val, true)
+	return validatePath(val, volumeMode)
 }
 
-func validatePath(val string, validateMountMode bool) (string, error) {
+func validatePath(val string, validateMode int) (string, error) {
```

Comment:
@calavera @vdemeester I agree pass validator function is better, so I added a commit to change `volume.ValidateMountMode` a bit, what about now?